components/nav/nav.sass

```diff
@@ -14,48 +15,38 @@
 .nav--minimal
   height: 5vh
 
-  .nav__logo
-    height: 0
-
-  .nav__content
-    height: 100%
-
-.nav__logo
-  height: 50%
-  transition: .3s height
-
 .nav__content
-  height: 50%
-  width: 50%
+  height: 100%
+  width: 65%
   display: flex
   align-items: center
   justify-content: space-between
   transition: .3s height
+  font-size: 1.3rem
 
 .nav__link
   text-decoration: none
-  color: $light-neutral-color
+  color: $dark-neutral-color
   transition: .3s color
 
 .nav__link:hover 
-  color: white
-
+  color: black
+    
 .nav__button
-  background: $light-neutral-color
+  background: $nav-button-background-color
   color: $dark-neutral-color
   text-decoration: none
   padding: 0.5rem 1.5rem
   border-radius: 50px
   font-size: 1rem
-  font-weight: 400
-  transition-property: transform, background, color
+  transition-property: transform, box-shadow
   transition-duration: .3s
+  border: 1px solid rgba($dark-neutral-color, 0.5)
 
-.nav__button:hover
-  background: white
-  color: black
+.nav__button:hover 
   transform: translateY(-2px)
-  cursor: pointer
+  box-shadow: 0 2px 2px rgba(0, 0, 0, 0.3)
```

Comment:
Let's move this shadow style into a variable so we can re-use it, that will keep our shadows consistent.